lib/CL/pocl_llvm_wg.cc

```diff
@@ -762,6 +762,8 @@ void *pocl_llvm_create_context_for_program(char *ProgramBcContent,
   // parse the program's bytes into a llvm::Module
   if (P == nullptr ||
       !P->init(ProgramBcContent, ProgramBcSize, TempBitcodePath)) {
+    if (P)
```

Comment:
Please use the explicit comparison style P != nullptr for pointer checks (just for style uniformity).